### DISCUSSION TITLE
V23Wayland模式启动器全屏bug

### DISCUSSION THREAD

**Discussion by Aninana** [linuxdeepin/developer-center#4477] [Troubleshooting | 故障排查]
原帖：https://bbs.deepin.org/phone/zh/post/257128
Wayland模式下，启动器全屏，放入二级菜单中的图标无法取出，如图：
（版本好像是Beta版）
![20230523221450799_截图_选择区域_20230523221042](https://github.com/linuxdeepin/developer-center/assets/130546027/745bbfe7-1740-4dd9-b6ba-79ff258afda8)

**Comment by BLumia** [comment]
意思是 x11 下正常，仅 wayland 有问题？这个不太像是 wayland 仅有的问题的样子。实际现象是尝试往外拖拖不出来吗？

**Reply by Aninana** [reply]
是的，实测X11下正常，Wayland下是这个情况：
![截图_VirtualBox Machine_20230524085959](https://github.com/linuxdeepin/developer-center/assets/130546027/362d1c0b-12ab-4c61-9729-1dc8983e8472)
